### ISSUE TITLE
No sound

### ISSUE THREAD

**Issue by project-cafe** [wh0/bookconfig#4]
Unsure what is the problem here, but I've noticed no sound. CMUS doesn't recognise any output devices. I have read that these things use the VT1613 chip, and maybe this doesn't include the driver or something. (Just a warning if you do respond with troubleshooting steps, I'm quite a Linux noob so you may need to dumb anything down a little.)

**Comment by wh0** [comment]
do you know if the upstream project had sound working?

**Comment by project-cafe** [comment]
I do not, I'm really new to this, I didn't remember sound working on the stretch repo from someone else based on this, I was hoping this would have sound working.

**Comment by wh0** [comment]
could you ask about it on their mailing list, https://groups.google.com/forum/?fromgroups#!forum/vt8500-wm8505-linux-kernel

**Comment by project-cafe** [comment]
I've taken a quick look through that, and the last legitimate post seems to be from last year, and for some reason when trying to make a post it just, loads forever.

**Comment by wh0** [comment]
indeed, I'm not seeing a new post from you there. not sure how to get in contact with them then

**Comment by project-cafe** [comment]
i do think ive seen some posts online about vt1613 support? but as far as i know the posts are like around 10 years old so not sure how much help theyd be

**Comment by wh0** [comment]
do they have instructions? I don't actually know how to set up a device with the device tree system, I've only rebased existing work

**Comment by project-cafe** [comment]
i honestly dont remember, but its worth a look. iirc i just googled "vt1613 linux" or something
